fdbcli/MetaclusterCommands.actor.cpp

```diff
@@ -634,7 +654,8 @@ std::vector<const char*> metaclusterHintGenerator(std::vector<StringRef> const&
 	} else if (tokencmp(tokens[1], "register") && tokens.size() < 5) {
 		static std::vector<const char*> opts = { "<NAME>",
 			                                     "connection_string=<CONNECTION_STRING>",
-			                                     "[max_tenant_groups=<NUM_GROUPS>]" };
+			                                     "[max_tenant_groups=<NUM_GROUPS>]",
+			                                     "[auto_tenant_assignment=<enabled|disabled>]" };
```

Comment:
The pattern here is different than the one below for configure. Maybe we should converge these around the configure example, so you would have:

`[max_tenant_groups=<NUM_GROUPS> | auto_tenant_assignment=<enabled|disabled>]`

And then you would return:

```
return std::vector<const char*>(opts.begin() + std::min<int>(2, tokens.size()) - 2, opts.end());
```

Reply:
Done. Is there code comment about the in/out/return of these `XXHintGenerator` functions?